<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="0" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-10T20:22:36 IST" name="Default suite" finished-at="2025-04-10T20:23:12 IST" duration-ms="35942">
    <groups>
    </groups>
    <test started-at="2025-04-10T20:22:36 IST" name="Default test" finished-at="2025-04-10T20:23:12 IST" duration-ms="35942">
      <class name="TestNg.CreatecampaignTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:36 IST" name="beforeSuite" finished-at="2025-04-10T20:22:36 IST" duration-ms="22" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:36 IST" name="beforeTest" finished-at="2025-04-10T20:22:36 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:36 IST" name="beforeClass" finished-at="2025-04-10T20:22:39 IST" duration-ms="2647" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:39 IST" name="beforeMethod" finished-at="2025-04-10T20:22:41 IST" duration-ms="2277" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="CreatecampaignTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:41 IST" name="CreatecampaignTest" finished-at="2025-04-10T20:22:48 IST" duration-ms="6832" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[im faling the script]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: im faling the script
at org.testng.Assert.fail(Assert.java:111)
at TestNg.CreatecampaignTest.CreatecampaignTest(CreatecampaignTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreatecampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:48 IST" name="afterMethod" finished-at="2025-04-10T20:22:49 IST" duration-ms="1016" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:49 IST" name="beforeMethod" finished-at="2025-04-10T20:22:49 IST" duration-ms="581" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="CreatecampaignTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:49 IST" name="CreatecampaignTest" finished-at="2025-04-10T20:22:55 IST" duration-ms="5749" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[im faling the script]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: im faling the script
at org.testng.Assert.fail(Assert.java:111)
at TestNg.CreatecampaignTest.CreatecampaignTest(CreatecampaignTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreatecampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:55 IST" name="afterMethod" finished-at="2025-04-10T20:22:56 IST" duration-ms="916" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:56 IST" name="beforeMethod" finished-at="2025-04-10T20:22:57 IST" duration-ms="620" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="CreatecampaignTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:22:57 IST" name="CreatecampaignTest" finished-at="2025-04-10T20:23:02 IST" duration-ms="5688" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[im faling the script]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: im faling the script
at org.testng.Assert.fail(Assert.java:111)
at TestNg.CreatecampaignTest.CreatecampaignTest(CreatecampaignTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreatecampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:02 IST" name="afterMethod" finished-at="2025-04-10T20:23:03 IST" duration-ms="1107" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:03 IST" name="beforeMethod" finished-at="2025-04-10T20:23:06 IST" duration-ms="2330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="CreatecampaignTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:06 IST" name="CreatecampaignTest" finished-at="2025-04-10T20:23:10 IST" duration-ms="4448" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[im faling the script]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: im faling the script
at org.testng.Assert.fail(Assert.java:111)
at TestNg.CreatecampaignTest.CreatecampaignTest(CreatecampaignTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreatecampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:10 IST" name="afterMethod" finished-at="2025-04-10T20:23:12 IST" duration-ms="1235" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:12 IST" name="afterClass" finished-at="2025-04-10T20:23:12 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:12 IST" name="afterTest" finished-at="2025-04-10T20:23:12 IST" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="AfterSuite()[pri:0, instance:TestNg.CreatecampaignTest@6caf0677]" started-at="2025-04-10T20:23:12 IST" name="AfterSuite" finished-at="2025-04-10T20:23:12 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterSuite -->
      </class> <!-- TestNg.CreatecampaignTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
